### ISSUE TITLE
Feature request: check groups

### ISSUE THREAD

**Issue by cloudunboxed-olorinhenderson** [Cyclenerd/static_status#21]
First of all, thank you so much for this great project! :)

Currently, all of the status checks are grouped into two categories in the HTML - Operational and Outage. I have a large number of checks to add, and it would be really helpful if we could segregate the checks into custom group names.

For example, instead of just having Operational, we'd have another green list header with a custom "My list 1" name containing a number of uptme checks, and then beneath it, another table list called "My list 2" with more checks inside etc.

Thank you in advance for any help you can offer with this request. :)

**Comment by Cyclenerd** [comment]
good idea. but it is a major change.

**Comment by cloudunboxed-olorinhenderson** [comment]
Yeah I looked at the code and saw that. I made it work for me at the output stage of the operational checks, by adding If statements that look for test names. It's gross, but it works for me! It'd be great to get proper support though. That's the only thing lacking of this beauty :)

**Comment by somera** [comment]
The idea sound good!

**Comment by brknkfr** [comment]
I tried something like a "group" function in this pull request: https://github.com/Cyclenerd/static_status/pull/36
